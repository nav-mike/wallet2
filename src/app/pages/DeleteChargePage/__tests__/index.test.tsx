import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { DeleteChargePage } from '..';

const shallowRenderer = createRenderer();

describe('<DeleteChargePage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<DeleteChargePage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
