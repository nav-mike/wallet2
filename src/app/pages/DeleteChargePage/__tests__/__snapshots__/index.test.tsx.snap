// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeleteChargePage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Delete charge
    </title>
    <meta
      name="Delete charge page"
    />
  </Helmet>
  <NavBar />
  <PageWrapper>
    <FlexContainer>
      <ColumnsContainer>
        <H1>
          Delete charge
        </H1>
        <SearchForm />
        <ChargesTable
          charges={
            Array [
              Object {
                "date": 1995-12-17T01:24:00.000Z,
                "id": "123-456-789",
                "type": "food",
                "value": 12.23,
              },
            ]
          }
        />
      </ColumnsContainer>
    </FlexContainer>
  </PageWrapper>
</React.Fragment>
`;
